lib/ui/homePage(temporary).dart

```diff
@@ -7,7 +7,7 @@ import 'package:google_fonts/google_fonts.dart';
 class MyHomePage extends StatefulWidget {
   final String title = "hello";
   final String name;
-  MyHomePage(this.name);
+  MyHomePage([this.name]);
```

Comment:
🤔

Reply:
using it as optional parameter.

Reply:
But do we have to?

Reply:
So i should delete them along with snack bar..